src/main/resources/assets/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -20,11 +20,20 @@ end
 
 local function get(paste)
     write( "Connecting to pastebin.com... " )
+    -- Add a cache buster so that spam protection is re-checked
+    local cacheBuster = tostring(math.random())
```

Comment:
Can we make this `("%x"):format(math.random(0, 2^30))` instead? I realise nobody will ever see this, but just feels a little nicer.